pkg/grafana/rules-handler.go

```diff
@@ -21,7 +22,7 @@ func NewRuleHandler(provider Provider) *RuleHandler {
 
 // Kind returns the name for this handler
 func (h *RuleHandler) Kind() string {
-	return "PrometheusRules"
+	return "PrometheusRuleGroup"
```

Comment:
Why prefix this with `Prometheus` and not Dashboard/Datasource with `Grafana`?

Reply:
Because "Dashboard" is obvious whereas "rule" could be anything? Rule is potentially more ambiguous. I suspect that naming is something that could come up in wider discussions.